Logger.Tests/LogFactoryTests.cs

```diff
@@ -6,6 +6,42 @@ namespace Logger.Tests
     [TestClass]
     public class LogFactoryTests
     {
-        
+
+        [TestMethod]
+        public void CreateLogger_FileLoggerFailsConfiguration_ReturnNull()
+        {
+            // Arrange
+            LogFactory logFac = new LogFactory();
```

Comment:
```suggestion
            LogFactory logFactory = new LogFactory();
```
Avoid using abbreviated names.